src/components/LocksCard.tsx

```diff
@@ -29,9 +29,8 @@ import { TrackDisplay } from './TrackDisplay'
 export const LocksCard = () => {
   const [currentBlock, setCurrentBlock] = useState(0)
   const [expectedBlockTime, setExpectedBlockTime] = useState(0)
-  const { api } = useNetwork()
+  const { api, assetInfo, trackList } = useNetwork()
```

Comment:
with TrackDisplay there's no need from the list any more
```suggestion
  const { api, assetInfo } = useNetwork()
```